kp/config.go

```diff
@@ -0,0 +1,9 @@
+package kp
+
+type Config struct {
+	KafkaConfig KafkaConfig `json:"kafka"`
+}
+
+type KafkaConfig struct {
+	KafkaBootstrap string `env:"CONFIG__KAFKA__BOOTSTRAP" default:"localhost:9092" json:"kafka_bootstrap"`
```

Comment:
isn't this a library like? I don't think it should read config like this, I think it should have asked for connection instead, no?